lib/kits/core-ui/paged-publication.js

```diff
@@ -168,10 +188,15 @@ const PagedPublication = (scriptEl, mainContainer = '') => {
         const shoppingBtn = options.el.querySelector('.sgn__offer-shopping');
 
         if (
-            scriptEls.offerClickBehavior === 'open-offer-link' &&
+            scriptEls.offerClickBehavior === 'open_webshop_link_in_tab' &&
             hotspot?.webshop
         ) {
             window.open(hotspot.webshop);
+        } else if (
+            scriptEls.offerClickBehavior === 'redirect_to_webshop_link' &&
+            hotspot?.webshop
+        ) {
+            window.open(hotspot.webshop, '_self');
```

Comment:
How is this different from the ordinary `location.href = hotspot.webshop;`?

Reply:
Nothing really. But I just read that location.href might be better so I'll change it to that :D